runner/Dockerfile

```diff
@@ -1,21 +1,10 @@
-FROM ubuntu:18.04
+FROM debian:9-slim
 
-RUN apt-get update && \
-  apt-get install -y \
-  linux-tools-4.15.0-22-generic git \
-  # Bitcoin Core dependencies
-  clang build-essential libtool autotools-dev \
-  automake pkg-config libssl-dev libevent-dev bsdmainutils \
-  libprotobuf-dev \
-  protobuf-compiler libboost-system-dev libboost-filesystem-dev \
-  libboost-chrono-dev libboost-program-options-dev libboost-test-dev \
-  libboost-thread-dev \
-  # run_bench.py dependencies
-  curl wget time python3-dev python3-pip && \
-  apt-get autoremove
-                
+COPY provision /usr/bin/provision
+RUN /usr/bin/provision
```

Comment:
@MarcoFalke this addresses your concern from #25.

Reply:
Is this file checked into git?

Reply:
Argh; apologies, that file appears in https://github.com/chaincodelabs/bitcoin-perfmonitor/pull/26/files#diff-48bd6c8b60cff4844a8626bd326aab5a.